addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4192,9 +4243,25 @@ registry.ImageTools = SnippetOptionWidget.extend({
             } else {
                 return '';
             }
+        } else if (methodName === 'transform') {
+            return this._isTransformed() ? 'true' : '';
+        } else if (methodName === 'crop') {
+            return this._isCropped() ? 'true' : '';
         }
         return this._super(...arguments);
     },
+    /**
+     * @override
+     */
+    _computeWidgetVisibility(widgetName, params) {
+        if (params.optionsPossibleValues.resetTransform) {
+            return this._isTransformed();
+        } 
+        if (params.optionsPossibleValues.resetCrop) {
+            return this._isCropped();
+        } 
```

Comment:
@younn-o Could you configure your editor to strip end-of-line spaces please (that's normally the case by default)? The line I am commenting here (merged) has one (and not the only one) and it's really a pain to work in this file because of it now. And that damn GitHub is not showing them :shrug: 

Could you also make a PR to remove them where you introduced them please? Would be better that having them removed in all commits I am reviewing touching this file :wink:

Reply:
Sorry about that ! The PR is here : https://github.com/odoo/odoo/pull/74804